.openpublishing.redirection.json

```
@@ -1904,6 +1904,10 @@
             "source_path": "docs/csharp/programming-guide/classes-and-structs/classes.md",
             "redirect_url": "/dotnet/csharp/fundamentals/types/classes"
         },
+        {
+            "source_path": "/dotnet/csharp/programming-guide/classes-and-structs/destructors",
+            "redirect_url": "docs/csharp/programming-guide/classes-and-structs/finalizers.md"
+        },
```

It should look like this:

```suggestion
        {
            "source_path": "/docs/csharp/programming-guide/classes-and-structs/destructors.md",
            "redirect_url": "/dotnet/csharp/programming-guide/classes-and-structs/finalizers"
        },
```

---

Except without the leading forward slash on line 1908.

---

Fixed. Great :eyes: @gewarren